### PR TITLE
Added Defold code examples for the Authentication page

### PR DESCRIPTION
**Pull request by britzl**

### CODE REVIEW

File: docs/authentication.md [hunk 1]
```diff
@@ -554,6 +594,22 @@ You can optionally import Facebook friends into Nakama's [friend graph](social-f
     }
     ```
 
+=== "Defold"
+    ```lua
+    -- Use the official Defold Facebook integration (www.defold.com/extension-facebook)
+    local permissions = { "public_profile" }
+    facebook.login_with_permissions(permissions, facebook.AUDIENCE_EVERYONE, function(self, data)
```
Comment (lugehorsam) [inline docs/authentication.md:601]: could you add a comment on what the facebook audience constant does and whether a certain value is required?

**Comment by lugehorsam** [comment]
One minor comment otherwise LGTM 👍 

Are there plans to add Defold examples for the other functionality?

File: docs/authentication.md (hunk 1 above)
Comment (britzl) [reply docs/authentication.md]: Sure, thing. I updated the code snippet and set audience to NONE (since the example only requests read permissions)

**Comment by britzl** [comment]
> Are there plans to add Defold examples for the other functionality?

Yes. We will create additional PRs for the other docs. It was a little bit more time consuming than I first thought :-)

**Comment by lugehorsam** [comment]
👍 thanks

**Pull request merged**